chainio/txmgr/txmgr.go

```diff
@@ -117,7 +120,7 @@ func (m *SimpleTxManager) Send(ctx context.Context, tx *types.Transaction) (*typ
 
 	tx, err = contract.RawTransact(opts, tx.Data())
 	if err != nil {
-		return nil, fmt.Errorf("send: failed to send txn: %w", err)
+		return nil, sdktypes.WrapError(fmt.Errorf("send: tx %v failed.", tx.Hash().String()), err)
```

Comment:
@shrimalmadhur added this to help with debugging. See https://github.com/Layr-Labs/avs-sync/issues/13